docker/docker.go

```diff
@@ -164,7 +164,7 @@ func DoRollingUpdateContainer(ctx context.Context, dClient *client.Client, image
 		return fmt.Errorf("[%s] Failed rolling update of container: docker client is nil for container [%s] on host [%s]", plane, containerName, hostname)
 	}
 	logrus.Debugf("[%s] Checking for deployed [%s]", plane, containerName)
-	isRunning, err := IsContainerRunning(ctx, dClient, hostname, containerName, false)
+	isRunning, err := DoesContainerExist(ctx, dClient, hostname, containerName, false)
```

Comment:
It is better to make the variable name matches the actual meaning to reduce confusion. 
```suggestion
	exist, err := DoesContainerExist(ctx, dClient, hostname, containerName, false)
```